mlir/lib/Target/CppOutput/gridwise_convolution_implicit_gemm_v4r4.cpp

```diff
@@ -181,6 +181,7 @@ static constexpr StringLiteral kCppEpiloguePart2Format = R"(
 )";
 
 static constexpr StringLiteral kGemmNameABlockCopySrcDataPerRead[] = {
```

Comment:
I thought we should not touch cpp backend for new development. Did you have the change verified in MIOpen scope?

Reply:
we use gemmG when MIOpen xdlops, but non-xdlops kernel don't contain gemmG
If we want to support group for non-xdlops, the kernel must contains gemmG .
you can't see gemmG  in MIOpen non xdlops kernel due to:

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/087b5cc995a9bd8a7cbbdda93d0f3aca80cddd0b/src/solver/conv_hip_implicit_gemm_fwd_v4r4.cpp#L610

Reply:
@asleepzzz / @jerryyin Let's discuss more in the meeting later today in US time. But I think the general direction is to keep C++ path as-is, by *NOT* supporting group convolution.

Reply:
@whchung Yes, I think the link further supports the idea that we should not support group conv in cpp path considering that it will not be natively supported in MIOpen + we have decided to not adding new feature to cpp path. That leads to reverting changes under `mlir/lib/Target/CppOutput/` directory.

Reply:
ok, but group already in MIOpen xdlops
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/087b5cc995a9bd8a7cbbdda93d0f3aca80cddd0b/src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp#L917

Reply:
@asleepzzz C++ path in this project is just experimental, so we don't want to add features which nobody would use. The real meat is in the ISA path where we directly generate GCN ISA.

Reply:
remove it